kodi_addon_checker/check_addon.py

```diff
@@ -345,18 +345,34 @@ def _check_complex_addon_entrypoint(report: Report, addon_path, parsed_xml, max_
 
         if library:
             filepath = os.path.join(addon_path, library)
+            name, ext = os.path.splitext(filepath)
 
             if not os.path.isdir(filepath):
 
-                if os.path.exists(filepath):
+                if os.path.exists(filepath) and ext == '.py':
                     number_of_lines(report, filepath, library,
                                     max_entrypoint_line_count)
-
+                elif ext == '.xml':
+                    number_of_lines_for_xml(report, filepath, library,
```

Comment:
I don't think we care about the size of an xml entry point at all. So you can drop this branch of the logic

Reply:
Done